apps/ts.c

```diff
@@ -980,6 +980,10 @@ static X509_STORE *create_cert_store(const char *CApath, const char *CAfile,
     const char *propq = app_get0_propq();
 
     cert_ctx = X509_STORE_new();
+    if (cert_ctx == NULL){
+        BIO_printf(bio_err, "X509_STORE_new failure\n");
```

Comment:
Maybe `memory allocation failure\n` would be sufficient.

Reply:
> Maybe `memory allocation failure\n` would be sufficient.

All fixed